### PR TITLE
PLAP-1458 Added optional filter budget tag for transaction-service apis

### PR DESCRIPTION
**Pull request by nstergiou-prodyna**

### CODE REVIEW

File: openapi-docs/oneapp-transaction-api-service.yaml [hunk 1]
```diff
@@ -154,6 +154,9 @@ components:
           type: string
         categoryCode:
           type: string
+        budgetTag:
+          type: string
+          description: one of needs, wants, savings, other
```
Comment (cmanios) [inline openapi-docs/oneapp-transaction-api-service.yaml:159]: Hi @nstergiou-prodyna ,

Maybe add also :

```yaml
enum:
- needs
- wants
- savings
- other
```

? Thanks!

File: openapi-docs/oneapp-transaction-service.yaml [hunk 2]
```diff
@@ -366,6 +366,9 @@ components:
           type: string
         categoryCode:
           type: string
+        budgetTag:
+          type: string
+          description: one of needs, wants, savings, other
```
Comment (cmanios) [inline openapi-docs/oneapp-transaction-service.yaml:371]: Hi @nstergiou-prodyna ,

Maybe add also :

```yaml
enum:
- needs
- wants
- savings
- other
```

? Thanks!

**Review by cmanios — APPROVED** [review]

File: openapi-docs/oneapp-transaction-api-service.yaml (hunk 1 above)
Comment (nstergiou-prodyna) [reply openapi-docs/oneapp-transaction-api-service.yaml]: Good point, done

File: openapi-docs/oneapp-transaction-service.yaml (hunk 2 above)
Comment (nstergiou-prodyna) [reply openapi-docs/oneapp-transaction-service.yaml]: Good point, done

**Pull request merged**